services/core-web/src/components/Forms/permits/PermitAmendmentSecurityForm.js

```diff
@@ -56,7 +57,7 @@ export const PermitAmendmentSecurityForm = (props) => (
           title="Are you sure you want to cancel?"
           okText="Yes"
           cancelText="No"
-          disabled={props.submitting}
+          onConfirm={() => props.onCancel()}
```

Comment:
We should keep the disabled line in still? That way they can't cancel out when the form is submitting

Reply:
Its still on the cancel button, I just removed it from the confirm actions
`<Button className="full-mobile" type="secondary" disabled={props.submitting}>
            Cancel
          </Button>`